contrib/scripts/kind.sh

```diff
@@ -9,7 +9,15 @@ default_workers=1
 default_cluster_name=""
 default_image=""
 default_kubeproxy_mode="iptables"
-default_ipfamily="dual"
+if [ "$(uname 2>/dev/null)" == "Linux" ] ; then
```

Comment:
Could simplify the conditions to (pseudo code) `if LINUX && $(sysctl -n net.ipv6.conf.all.disable_ipv6) == 1` then `ipv4` else `dual`?

Reply:
addressed in https://github.com/cilium/cilium/pull/30332/commits/40b2ec6716eb3d83ef727d75084c35089b1d461b